### PR TITLE
zh-cn: Format /mdn using Prettier

### PR DESCRIPTION
**Pull request by queengooborg**
This PR formats the `/mdn` folder in the Simplified Chinese locale using Prettier.

### CODE REVIEW

File: files/zh-cn/mdn/writing_guidelines/howto/markdown_in_mdn/index.md [hunk 1]
```diff
@@ -411,19 +414,19 @@ const greeting = "I'm a bad example";
...
 | cell 4    | cell 5    | cell 6    |
 ```
 
 而不是这种风格：
 
 ```md
-Heading 1 | Heading 2 | Heading 3
- --- | --- | ---
+| Heading 1 | Heading 2 | Heading 3 |
+| --------- | --------- | --------- |
```
Comment (yin1999) [inline files/zh-cn/mdn/writing_guidelines/howto/markdown_in_mdn/index.md:426]: This is a bad-example here, we should add `-nolint` :)

I've created a PR to revert the changes in en-US: mdn/content#27978

File: files/zh-cn/mdn/writing_guidelines/howto/markdown_in_mdn/index.md [hunk 2]
```diff
@@ -515,7 +526,7 @@ GFM 表格语法的主要限制是：
 因为它们有一个标题列，GFM 无法表示这些页面。因此，应该使用 HTML。为了获得特殊的样式，还需要将 `"properties"` 类应用于表格：
 
 ```html
-<table class="properties">
+<table class="properties"></table>
```
Comment (yin1999) [inline files/zh-cn/mdn/writing_guidelines/howto/markdown_in_mdn/index.md:529]: also `nolint` here

File: files/zh-cn/mdn/writing_guidelines/howto/markdown_in_mdn/index.md (hunk 1 above)
Comment (queengooborg) [reply files/zh-cn/mdn/writing_guidelines/howto/markdown_in_mdn/index.md:426]: I'm down to add `-nolint` here for now until the upstream version's updated and we resync with upstream!

File: files/zh-cn/mdn/writing_guidelines/howto/markdown_in_mdn/index.md (hunk 2 above)
Comment (queengooborg) [reply files/zh-cn/mdn/writing_guidelines/howto/markdown_in_mdn/index.md:529]: I noticed that upstream also had a `-nolint`, but it doesn't feel like the proper use of `-nolint`, so I'm removing it from upstream and formatting the code block like this in the English content!

**Review by yin1999 — APPROVED** [review]
I've modify the `markdown_in_mdn/index.md`, just following the upstream: https://github.com/mdn/content/blob/1e207a5328e8158a145004fa8ee34783178347c5/files/en-us/mdn/writing_guidelines/howto/markdown_in_mdn/index.md?plain=1#L444C7-L458

Thanks @queengooborg
